docs/data/rpc/admin-guide.mdx

```diff
@@ -287,13 +287,24 @@ HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_003/{
 
 3. Run the image, making sure to [mount a volume](https://docs.docker.com/storage/volumes/) to your container where the above configuration is stored. An example for Testnet:
 
+If you are running locally on a macOS for instance, you could create a volume in your home directory: `~/test-rpc-config`
+
+Then you would add the following config files to that local directory:
+
+- soroban-rpc-config.toml
+- stellar-captive-core.cfg
+
+Then you would mount that volume using the following parameter: `-v /Users/<USERNAME>/test-rpc-config:/opt/stellar`.
+
+Your running container would mount that volume at the path `/opt/stellar`
+
+In that example, you could run the `stellar/stellar-rpc` container with the following command:
+
```

Comment:
I like having these instructions here a lot! I think it would probably make more sense to split "step 3" into two distinct steps:

3. do things to create the mount-able volume
4. run the image, pointing to the directory from step 3

Reply:
Yea absolutely.  You can mount an existing volume with -v with the files pre-populated but that's not as clear as separating it into 2 steps.